extensions/adobe/experience/offer-management/content/email-content.schema.json

```diff
@@ -0,0 +1,47 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id":
+    "https://ns.adobe.com/experience/offer-management/content/email-content",
+  "type": "object",
+  "title": "Content",
+  "description": "E-mail offer content",
+  "definitions": {
+    "email-content": {
+      "type": "object",
+      "title": "E-mail",
+      "description": "E-mail properties",
+      "properties": {
+        "@type": {
+          "type": "string",
+          "const": "https://ns.adobe.com/experience/offer-management/content/email-content"
+        },
+        "xdm:from": {
+          "type": "string",
```

Comment:
It might make sense to split this into two properties. One with the full text of the `from` field, and another one with the parsed email address.

Reply:
`email-content` is gone